tests/test_finetune.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import timm
+import torch
+import importlib
+import terratorch 
+import subprocess
+import os 
+
+from terratorch.models.backbones.prithvi_vit import checkpoint_filter_fn as checkpoint_filter_fn_vit
+from terratorch.models.backbones.prithvi_swin import checkpoint_filter_fn as checkpoint_filter_fn_swin
+
+@pytest.mark.parametrize("model_name", ["prithvi_swin_B", "prithvi_swin_L", "prithvi_vit_100", "prithvi_vit_300"])
+def test_finetune_multiple_backbones(model_name):
+
+    # Instantiating and creating a manufactured 
+    # checkpoint just to test the finetuning pipeline
+    if "vit" in model_name :
+        module_str = "terratorch.models.backbones.prithvi_vit"
+        ckpt_filter = checkpoint_filter_fn_vit
+    elif "swin" in model_name:
+        module_str = "terratorch.models.backbones.prithvi_swin"
+        ckpt_filter = checkpoint_filter_fn_swin
+
+    module_instance = importlib.import_module(module_str)
+
+    model_template = getattr(module_instance, model_name)
+
+    model_instance = model_template()
```

Comment:
Instead of this, can we leverage `timm.create_model` to directly create the backbone from the string?

This wont work for all models, as not all of them support being created from timm, but at least for this I think its a better solution. I worry about the if statement of import module paths growing and becoming unwieldy.

Reply:
Considering the way the code is currently structured and the models available on terratorch, it seems to be enough for now, but we could do it.

Reply:
Since its a relatively quick code change and results in cleaner code I think we should

Reply:
It's done now.